app/build.gradle

```diff
@@ -93,6 +93,8 @@ android {
     // See https://developer.android.com/jetpack/docs/release-notes#september_19_2018
     packagingOptions {
         exclude 'META-INF/proguard/androidx-annotations.pro'
+        // Remove when https://github.com/mozilla-mobile/android-components/issues/2900 is addressed
```

Comment:
nit: can we file our own follow-up issue that tracks the a-c issue and reference that instead? It makes it harder to forget to fix.

Reply:
Sounds good